### PR TITLE
New version: DigiDNA.iMazing version 2.14.7

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57488)

### CODE REVIEW

File: manifests/d/DigiDNA/iMazing/2.14.7/DigiDNA.iMazing.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: DigiDNA.iMazing
+PackageVersion: 2.14.7
```
Comment (ItzLevvie) [inline manifests/d/DigiDNA/iMazing/2.14.7/DigiDNA.iMazing.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.14.7.0
```

File: manifests/d/DigiDNA/iMazing/2.14.7/DigiDNA.iMazing.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: DigiDNA.iMazing
+PackageVersion: 2.14.7
```
Comment (ItzLevvie) [inline manifests/d/DigiDNA/iMazing/2.14.7/DigiDNA.iMazing.locale.zh-CN.yaml:5]:
```suggestion
PackageVersion: 2.14.7.0
```

File: manifests/d/DigiDNA/iMazing/2.14.7/DigiDNA.iMazing.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: DigiDNA.iMazing
+PackageVersion: 2.14.7
```
Comment (ItzLevvie) [inline manifests/d/DigiDNA/iMazing/2.14.7/DigiDNA.iMazing.yaml:5]:
```suggestion
PackageVersion: 2.14.7.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by zachcarp** [comment]
This is one of three that keep failing for internal failures. I'll pull the logs later today and see if I can figure out whats going wrong.

**Comment by zachcarp** [comment]
@vedantmgoyal2009 This is an internal failure. An upstream team sent us an unexpected value that we attempted to parse into an enumeration which failed. We are currently working on a fix for this. We will also build measures to help avoid this in the future.

**Pull request merged**